project-sftp-upload-of-qr-codes/axis_image_consumer.sh

```
@@ -0,0 +1,153 @@
+    curl_exit=$?
+
+    if [ $curl_exit -ne 0 ]; then
+        debug_log_file "Curl or base64 encoding failed with exit code $curl_exit"
+        return 1
+    fi
+
+    # Check if we got actual encoded data (should be more than 100 characters for a valid image)
+    data_size=${#encoded_data}
+    debug_log_file "Received $data_size characters of base64 encoded data"
+
+    if [ $data_size -lt 100 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)